sympy/physics/control/lti.py

```diff
@@ -976,10 +1007,247 @@ def is_biproper(self):
+        return self.args[-1].num_outputs
+
+    @property
+    def shape(self):
+        return self.args[-1].num_outputs, self.args[0].num_inputs
+
+    def doit(self, **kwargs):
+        """
+        Returns the resultant transfer function obtained after evaluating
+        the transfer functions in series configuration.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction, MIMOSeries, TransferFunctionMatrix
+        >>> tf1 = TransferFunction(a*p**2 + b*s, s - p, s)
+        >>> tf2 = TransferFunction(s**3 - 2, s**4 + 5*s + 6, s)
+        >>> tfm1 = TransferFunctionMatrix([[tf1, tf2], [tf2, tf2]])
+        >>> tfm2 = TransferFunctionMatrix([[tf2, tf1], [tf1, tf1]])
+        >>> MIMOSeries(tfm2, tfm1).doit()
+        TransferFunctionMatrix(((TransferFunction(2*(s**3 - 2)*(a*p**2 + b*s), (-p + s)*(s**4 + 5*s + 6), s), TransferFunction((-p + s)**2*(s**3 - 2)*(a*p**2 + b*s) + (-p + s)*(a*p**2 + b*s)**2*(s**4 + 5*s + 6), (-p + s)**3*(s**4 + 5*s + 6), s)), (TransferFunction((-p + s)*(s**3 - 2)**2*(s**4 + 5*s + 6) + (s**3 - 2)*(a*p**2 + b*s)*(s**4 + 5*s + 6)**2, (-p + s)*(s**4 + 5*s + 6)**3, s), TransferFunction(2*(s**3 - 2)*(a*p**2 + b*s), (-p + s)*(s**4 + 5*s + 6), s))))
+
+        """
+        _arg = (arg.doit()._expr_mat for arg in reversed(self.args))
+        res = MatMul(*_arg, evaluate=True)
+        return TransferFunctionMatrix.from_Matrix(res, self.var)
+
+    def _eval_rewrite_as_TransferFunctionMatrix(self, *args, **kwargs):
+        return self.doit()
+
+    def __add__(self, other):
+        if not isinstance(other, LinearTimeInvariant):
+             return NotImplemented
+
+        if self.is_SISO != other.is_SISO:
+            raise TypeError("Cannot add SISO and MIMO systems together.")
+
+        if isinstance(other, MIMOParallel):
+            arg_list = list(other.args)
+            return MIMOParallel(self, *arg_list)
+
+        return MIMOParallel(self, other)
+
+    __radd__ = __add__
+
+    def __sub__(self, other):
+        return self + (-other)
+
+    def __rsub__(self, other):
+        return -self + other
+
+    def __mul__(self, other):
+        if not isinstance(other, LinearTimeInvariant):
+             return NotImplemented
+
+        if other.is_SISO:
+            raise TypeError("Cannot multiply SISO and MIMO systems together.")
+
+        if isinstance(other, MIMOSeries):
+            self_arg_list = list(self.args)
+            other_arg_list = list(other.args)
+            return MIMOSeries(*other_arg_list, *self_arg_list)  # A*B = MIMOSeries(B, A)
+
+        arg_list = list(self.args)
+        return MIMOSeries(other, *arg_list)
+
+    def __neg__(self):
+        arg_list = list(self.args)
+        arg_list[0] = -arg_list[0]
+        return MIMOSeries(*arg_list)
+
+
+class Parallel(LinearTimeInvariant):
+    r"""
+    A class for representing parallel configuration of SISO and MIMO transfer functions.
```

Comment:
```suggestion
    A class for representing parallel configuration of SISO transfer functions.
```
this one doesn't support MIMO, right?